CITING.md

```diff
@@ -0,0 +1,94 @@
+# Citing Ginkgo                                           {#citing_ginkgo}
+
+The main Ginkgo paper describing Ginkgo's purpose, design and interface is
+available through the following reference:
+
+``` bibtex
+@misc{anzt2020ginkgo,
+    title={Ginkgo: A Modern Linear Operator Algebra Framework for High Performance Computing},
+    author={Hartwig Anzt and Terry Cojean and Goran Flegar and Fritz Göbel and Thomas Grützmacher and Pratik Nayak and Tobias Ribizel and Yuhsiang Mike Tsai and Enrique S. Quintana-Ortí},
+    year={2020},
+    eprint={2006.16852},
+    archivePrefix={arXiv},
+    primaryClass={cs.MS}
+}
+```
+
+Multiple topical papers exist on Ginkgo and its algorithms. The following papers
+can be used to cite specific aspects of the Ginkgo project.
+
+### On Portability
+
+``` bibtex
+@misc{tsai2020preparing,
+    title={Preparing Ginkgo for AMD GPUs -- A Testimonial on Porting CUDA Code to HIP},
+    author={Yuhsiang M. Tsai and Terry Cojean and Tobias Ribizel and Hartwig Anzt},
+    year={2020},
+    eprint={2006.14290},
+    archivePrefix={arXiv},
+    primaryClass={cs.MS}
+}
+```
+
+### On Software Sustainability
+
+``` bibtex
+@inproceedings{10.1145/3324989.3325719,
```

Comment:
This short form will be used by people to cite it in their LaTeX documents, right? So, is it possible to change this to something like 2020PascGinkgoBench or something like that ? 

The same for the references below.